tests/trainer/logging_/test_logger_connector.py

```diff
@@ -479,6 +479,27 @@ def test_step(self, *args, **kwargs):
         trainer.test(model)
 
 
+def test_can_return_tensor_with_more_than_one_element(tmpdir):
+    """Ensure {validation,test}_step return values are not included as callback metrics. #6623"""
+
+    class TestModel(BoringModel):
+
+        def validation_step(self, batch, *args, **kwargs):
+            return {"val": torch.tensor([0, 1])}
+
+        def test_step(self, batch, *args, **kwargs):
+            return {"test": torch.tensor([0, 1])}
+
+    model = TestModel()
+    model.validation_epoch_end = None
+    model.test_epoch_end = None
```

Comment:
do we already have a test to ensure that the step outputs are still returned to the `_epoch_end` hooks?

Reply:
Can this test be extended to make sure of it ?